analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -107,24 +107,21 @@ class Bundler : PackageManager() {
             installDependencies(workingDir)
 
             val (projectName, version, homepageUrl, declaredLicenses) = parseProject(workingDir)
+            val projectId = Identifier(javaClass.simpleName, "", projectName, version)
```

Comment:
Just use `toString()` instead of `javaClass.simpleName`.

Reply:
done